user_manual/universal_access.rst

```diff
@@ -0,0 +1,83 @@
+================
+Universal access
+================
+
+Universal access is very important to us. We follow web standards and check to
+make everything usable also with keyboard and assistive software such as
+screen readers. We aim to be compliant with the `Web Content Accessibility
+Guidelines 2.1 <https://www.w3.org/WAI/standards-guidelines/wcag/>`_
+on AA level, with the high contrast theme even on AAA level.
+We also follow the German BITV 2.0 guidelines.
+
+If you find any issues, don’t hesitate to report them on `our issue tracker
+<https://github.com/nextcloud/server/issues/>`_. And if you want to get
+involved, come `join our design team <https://nextcloud.com/design>`_!
+
+
+Zoom and responsiveness
+-----------------------
+
+The Nextcloud interface is fully responsive and usable on any size of screen.
+You can zoom in and out to fit the text and element size to your liking.
+The left navigation and right sidebar can be expanded or collapsed.
```

Comment:
Left and right are not to inclusive with RTL languages when they swap it at some point, is it needed?

Reply:
Good catch, fix is at https://github.com/nextcloud/documentation/pull/6433